### PR TITLE
Add clearsky cygnss layers

### LINKED ISSUE #3367: Add VIIRS ClearSky/CYGNSS WS SDR
| Layer Identifier | Title | Subtitle | Source Dataset(s) | Projection |Time Period|
| ------------- |-------------|-------------|------|------|--------|
|CYGNSS_L3_Wind_Speed_SDR_Daily|Wind Speed (Daily)|CYGNSS / DDMI|C1955682144-PODAAC|4326|2018-08-01 to Present|
|VIIRS_NOAA20_Clear_Sky_Confidence_Day|Clear Sky Confidence (Day)|NOAA-20 / VIIRS|CLDMSK_L2_VIIRS_NOAA20<br />CLDMSK_L2_VIIRS_NOAA20_NRT / v1|4326|2018-03-17 to Present|
|VIIRS_NOAA20_Clear_Sky_Confidence_Night|Clear Sky Confidence (Night)|NOAA-20 / VIIRS|CLDMSK_L2_VIIRS_NOAA20<br />CLDMSK_L2_VIIRS_NOAA20_NRT / v1|4326|2018-03-17 to Present|

### LINKED ISSUE #3368: Update SMAP SSS CAP Products
Both SMAP L3 Sea Surface Salinity CAP Monthly and 8Day-Running-Mean are upgrading from 4.3 to 5.0. The following are their 5.0 concept IDs:
| Layer Identifier | Concept ID |
| ------------- |-------------|
|SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean|C1972955240-PODAAC|
|SMAP_L3_Sea_Surface_Salinity_CAP_Monthly|C1972950629-PODAAC|

### LINKED ISSUE #3371: Remove TOPEX-Poseidon_JASON_Sea_Surface_Height_Anomalies_GDR_Cycles
PODAAC has asked us to remove TOPEX-Poseidon_JASON_Sea_Surface_Height_Anomalies_GDR_Cycles.

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #3367; Fixes #3368; Fixes #3371   .

[Description of the bug or feature]

- [x]  Adds 2 VIIRS NOAA-20 Clear Sky Confidence layers; clear sky descriptions for NOAA-20 and Suomi NPP
- [x] Adds 1 CYGNSS Wind Speed SDR layer
- [x] Updates descriptions for SMAP SSS CAP
- [x] Removes TOPEX-Posiedon_JASON_Sea_Surface_Height_Anomalies_GDR_Cycles
- [x] Updates MODIS BRDF Reflectance title to include NBAR

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/layers/cygnss/CYGNSS_L3_Wind_Speed_SDR_Daily.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+
+ TThe layer provides high resolution wind speed data that allows for global tropical ocean observation.  he primary mission objective for CYGNSS is to provide frequent space‐based measurements of ocean surface wind speed in the inner core of tropical cyclones. The orbital asynchronicity with respect to the local solar daytime/nightime patterns may provide additional scientific opportunity to study wind speed variations and patterns due to the diurnal cycle.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/cygnss/CYGNSS_L3_Wind_Speed_SDR_Daily.md:3]:
```suggestion
 The layer provides high resolution wind speed data that allows for global tropical ocean observation The primary mission objective for CYGNSS is to provide frequent space‐based measurements of ocean surface wind speed in the inner core of tropical cyclones. The orbital asynchronicity with respect to the local solar daytime/nighttime patterns may provide additional scientific opportunity to study wind speed variations and patterns due to the diurnal cycle.
```

File: config/default/common/config/metadata/layers/smap/SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
-SMAP Level 3 CAP sea surface salinity (SSS) standard mapped images contains 60-km spatial resolution SSS data averaged over a range of temporal scales. The selected dataset is the 8-day running mean JPL SMAP-SSS CAP version 4.2 product.
+The SMAP Sea Surface Salinity (Level 3, CAP, 8-Day Running Mean) layer is based on the Combined Active-Passive (CAP) retrieval algorithm developed at the Jet Populsion Laboratory (JPL) originally in the context of Aquarius/SAC-D and now extended to SMAP. JPL SMAP V5.0 SSS is based on the newly released SMAP V5 Level-1 Brightness Temperatures (TB).
```
Comment (edplato) [inline config/default/common/config/metadata/layers/smap/SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean.md:1]:
```suggestion
The SMAP Sea Surface Salinity (Level 3, CAP, 8-Day Running Mean) layer is based on the Combined Active-Passive (CAP) retrieval algorithm developed at the Jet Propulsion Laboratory (JPL) originally in the context of Aquarius/SAC-D and now extended to SMAP. JPL SMAP V5.0 SSS is based on the newly released SMAP V5 Level-1 Brightness Temperatures (TB).
```

**Review by edplato — CHANGES_REQUESTED** [review]
Looks good. I made some change suggestions that may need to be repeated in other sections.

File: config/default/common/config/metadata/layers/smap/SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean.md (hunk 2 above)
Comment (minniewong) [reply config/default/common/config/metadata/layers/smap/SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean.md:1]: 👀  good finds!

Comment (minniewong) [reply config/default/common/config/metadata/layers/smap/SMAP_L3_Sea_Surface_Salinity_CAP_8Day_RunningMean.md]: Fixed here and other related files.

**Comment by minniewong** [comment]
@edplato hope I caught it all!

**Review by edplato — APPROVED** [review]
🚀  Looks good!

**Pull request merged**